types/node/test/globals.ts

```diff
@@ -80,6 +80,11 @@ declare var RANDOM_GLOBAL_VARIABLE: true;
         dispatcher: undefined,
     });
 
-    // @ts-expect-error
-    NodeJS.fetch;
+    const reqinit: RequestInit = {};
+    reqinit.method; // $ExpectType string | undefined
+    const resinit: ResponseInit = {};
+    resinit.status; // $ExpectType number | undefined
+
+    const f: File = {} as any;
+    f.name; // $ExpectType string    
```

Comment:
```suggestion
    f.name; // $ExpectType string
```

Reply:
FWIW this would be fixed if you run `dprint fmt` 😄